### DISCUSSION TITLE
Why my token from auth "memberjwt" not invalid on "jwt" auth ?

### DISCUSSION THREAD

**Discussion by anggi117** [adonisjs/core#3802] [Help]
I have 2 auth JWT (jwt and memberjwt)
![image](https://user-images.githubusercontent.com/42998655/180150207-6782ccda-6119-4f10-9186-1d483007e68e.png)

and im already set the middleware to "jwt" in route.
![image](https://user-images.githubusercontent.com/42998655/180150233-e1536df7-a233-4951-923e-bc94fc0bdf96.png)

but why my token generate from "memberjwt" auth valid on auth "jwt" ?

is there any solution?

this my route using auth "memberjwt" and email its : [kotnol@gmail.com](mailto:kotnol@gmail.com)
![image](https://user-images.githubusercontent.com/42998655/180150434-953f1c6c-17ca-44f5-8d78-ff8965f42c01.png)

but im check it using auth.getuser() (authenticator: 'jwt' on app) its still showing with different data, even though that token from memberjwt
![image](https://user-images.githubusercontent.com/42998655/180150462-95be5561-22e6-4a53-ba9b-46e6c8d6e2e7.png)

i want that token from "memberjwt" auth its invalid in "jwt" auth

**Comment by dipedro** [comment]
You can create a new APP_KEY inside a .env file and change in config auth
